src/side-navigation/__integ__/side-navigation.test.ts

```diff
@@ -39,4 +39,16 @@ describe('SideNavigation', () => {
       })
     );
   });
+
+  describe('Items Header', () => {
+    it(
```

Comment:
This test is supposed to fail because the `#/light/side-navigation/integration` page does not, in fact, have the items header. It fails because of incorrectly written assertion on the line 50: the page.isExisting is asynchronous.

I propose to remove this test though because the logic is covered with unit tests. We usually create integration test for behaviours depending on browser in a way that cannot be well replicated in unit tests.

Nevertheless, it would be useful to add use the new property in one of the side navigation test pages just to showcase it and allow for quick manual checks.